### PR TITLE
Redux login flow

### PR DESCRIPTION
**Pull request by asafoj**

### CODE REVIEW

File: src/components/login/loginActions.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+function handleErrors(response) {
+  if (!response.ok) {
+      throw Error(response.statusText);
+  }
+  return response;
+}
+
+export function logInUser(email, password ) {
+  return (dispatch) => {
+    fetch("/login", {
```
Comment (chookie) [inline src/components/login/loginActions.js]: Fetch is nice but I think we need a polyfill as it does not work on IE or safari/webkit according to docs https://developer.mozilla.org/en/docs/Web/API/Fetch_API

Perhaps use https://github.com/github/fetch

File: src/components/login/LoginContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+import { logInUser, clearAlert } from './loginActions';
+import LoginForm from './components/loginForm/LoginForm.jsx';
+import './LoginContainer.scss';
+
+class LoginContainer extends Component {
+  static propTypes = {
+    logInUser: React.PropTypes.func.isRequired,
+    clearAlert: React.PropTypes.func,
+    errors: React.PropTypes.string,
```
Comment (chookie) [inline src/components/login/LoginContainer.jsx]: errors is not used anywhere.  The displayed message is hard coded.

File: src/components/login/LoginContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      password: ''
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.showAlert = this.showAlert.bind(this);
+    this.handleAlertDismiss = this.handleAlertDismiss.bind(this);
+  }
+
+  handleChange(event) {
```
Comment (asafoj) [inline src/components/login/LoginContainer.jsx]: Please se the link below about using arrow es6 class method declarations:

https://javascriptweblog.wordpress.com/2015/11/02/of-classes-and-arrow-functions-a-cautionary-tale/

File: src/components/login/loginActions.js (hunk 1 above)
Comment (asafoj) [inline src/components/login/loginActions.js]: I have replaced fetch this wit axios

**Pull request merged**